### PR TITLE
feat: create rlsr jso content

### PR DESCRIPTION
**Pull request by matthias-reis**
Bringing the rlsr.json content together. For other steps, it helps to have a named object as deps, instead of an array.

### CODE REVIEW

**Review by santiph — APPROVED** [review]

File: new/analyse/spec/create-rlsr-json-content.spec.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            {},
+            {},
+            {
+              currentVersion: '2.0.0',
+              dependsOn: [],
+            }
+          ),
+        },
+      };
+      result = await createRlsrJsonContent(mockedEnv);
```
Comment (santiph) [inline new/analyse/spec/create-rlsr-json-content.spec.ts:67]: `createRlsrJsonContent` is not an async function, why is this needed?

**Review by santiph — APPROVED** [review]

File: new/analyse/spec/create-rlsr-json-content.spec.ts (hunk 1 above)
Comment (matthias-reis) [reply new/analyse/spec/create-rlsr-json-content.spec.ts:67]: it returns a promise. I thought, it was the reason why a test failed (but probably wasn't). I change it (forgot about it).

Comment (santiph) [reply new/analyse/spec/create-rlsr-json-content.spec.ts:67]: It's because of the Module type, I guess.
It's typed to return either an Env or a Promise<Env> object.

I think I've always forced it as `result = createRlsrJsonContent(mockedEnv) as Env;`
